cmd/blockchaincmd/add_validator.go

```diff
@@ -548,9 +544,6 @@ func CallAddValidator(
 	if err != nil {
 		return err
 	}
-	if err = signatureaggregator.UpdateSignatureAggregatorPeers(app, network, extraAggregatorPeers, aggregatorLogger); err != nil {
```

Comment:
you need to call CreateSignatureAggregatorInstance if not exists somewhere in the code, all commands will need you to start one if not available